string psbmpc_mode
---
bool success
string status_msg
